src/System.Runtime.InteropServices/src/System.Runtime.InteropServices.csproj

```diff
@@ -19,28 +19,48 @@
     <Compile Include="$(CommonPath)\Interop\Windows\Ole32\Interop.CoGetStandardMarshal.cs">
       <Link>Common\Interop\Windows\Ole32\Interop.CoGetStandardMarshal.cs</Link>
     </Compile>
-    <Compile Include="System\Runtime\CompilerServices\Attributes.cs" />
-    <Compile Include="System\Runtime\InteropServices\Attributes.cs" />
-    <Compile Include="System\Runtime\InteropServices\ComAwareEventInfo.cs" />
-    <Compile Include="System\Runtime\InteropServices\HandleCollector.cs" Condition="'$(TargetsAOT)' != 'true'" />
-    <Compile Include="System\Runtime\InteropServices\IMarshal.cs" />
-    <Compile Include="System\Runtime\InteropServices\RuntimeEnvironment.cs" />
-    <Compile Include="System\Runtime\InteropServices\ComTypes\advf.cs" Condition="'$(TargetsAOT)' != 'true'" />
-    <Compile Include="System\Runtime\InteropServices\ComTypes\datadir.cs" Condition="'$(TargetsAOT)' != 'true'" />
-    <Compile Include="System\Runtime\InteropServices\ComTypes\dvaspect.cs" Condition="'$(TargetsAOT)' != 'true'" />
-    <Compile Include="System\Runtime\InteropServices\ComTypes\formatetc.cs" Condition="'$(TargetsAOT)' != 'true'" />
+    <Compile Include="System\Runtime\CompilerServices\IDispatchConstantAttribute.cs" />
+    <Compile Include="System\Runtime\CompilerServices\IUnknownConstantAttribute.cs" />
+    <Compile Include="System\Runtime\InteropServices\ComTypes\ADVF.cs" Condition="'$(TargetsAOT)' != 'true'" />
+    <Compile Include="System\Runtime\InteropServices\ComTypes\DATADIR.cs" Condition="'$(TargetsAOT)' != 'true'" />
+    <Compile Include="System\Runtime\InteropServices\ComTypes\DVASPECT.cs" Condition="'$(TargetsAOT)' != 'true'" />
+    <Compile Include="System\Runtime\InteropServices\ComTypes\FORMATETC.cs" Condition="'$(TargetsAOT)' != 'true'" />
     <Compile Include="System\Runtime\InteropServices\ComTypes\IAdviseSink.cs" Condition="'$(TargetsAOT)' != 'true'" />
     <Compile Include="System\Runtime\InteropServices\ComTypes\IDataObject.cs" />
     <Compile Include="System\Runtime\InteropServices\ComTypes\IEnumFormatETC.cs" Condition="'$(TargetsAOT)' != 'true'" />
     <Compile Include="System\Runtime\InteropServices\ComTypes\IEnumSTATDATA.cs" />
-    <Compile Include="System\Runtime\InteropServices\ComTypes\statdata.cs" Condition="'$(TargetsAOT)' != 'true'" />
-    <Compile Include="System\Runtime\InteropServices\ComTypes\stgmedium.cs" Condition="'$(TargetsAOT)' != 'true'" />
-    <Compile Include="System\Runtime\InteropServices\ComTypes\tymed.cs" Condition="'$(TargetsAOT)' != 'true'" />
+    <Compile Include="System\Runtime\InteropServices\ComTypes\STATDATA.cs" Condition="'$(TargetsAOT)' != 'true'" />
```

Comment:
I am a bit leery of this change. It should work without issue on non-Windows platforms, but on Windows since case is ignored we can hit issues with updating/syncing repos. In particular this can create a potentially invalid git repo on some hosting systems since the file will now have two names - `statdata.cs` and `STATDATA.cs`. For example see: https://developercommunity.visualstudio.com/content/problem/500562/failure-to-fork-httpsgithubcommonomonodevelop.html

Reply:
We have done similar case-only renames many times in the history of CoreFX repo. One more should not be a problem.

Reply:
This should also not be a problem on the git side, provided it was moved properly (`git mv` should generally do the trick).

Reply:
>This should also not be a problem on the git side, provided it was moved properly (git mv should generally do the trick).

@tannergooding Unfortunately that isn't true. That command doesn't change any of the underlying issues that exist with the naming issues in the issue I reported above. This isn't an issue with git itself, but rather a policy that can be applied on various hosting platforms (i.e. AzureDevOps).

@jkotas Yep, I just confirmed this repo has other instances of this and the DevDiv project in AzureDevOps fails to import it. Boo.

@hughbe In this case, looks like the repo is already poisoned with this problem. Resolving.